operator-common/src/main/java/io/strimzi/operator/common/MicrometerMetricsProvider.java

```diff
@@ -64,6 +65,7 @@ public Counter counter(String name, String description, Tags tags) {
     public Timer timer(String name, String description, Tags tags) {
         return Timer.builder(name)
                 .description(description)
+                .sla(Duration.ofMillis(10), Duration.ofMillis(25), Duration.ofMillis(50), Duration.ofMillis(100), Duration.ofMillis(500), Duration.ofMillis(1000), Duration.ofMillis(5000))
```

Comment:
I would not expect to see all these buckets with this level of granularity. I would say that everything under 500 or 1000 ms would be ok when the reconcile is fast without actually doing nothing or a few then we could have longer ones due to rolling pods, even over the minutes. Maybe 500, 1000, 30000, 60000, 120000, 240000 could have more sense? @scholzj wdyt?

Reply:
Well, it is hard. I guess different resources have different reconciliation times. I would maybe go for:
* 1s
* 5s
* 10s
* 30s
* 1m
* 2m
* 5m
* Inf

Reply:
5 min is our upper default limit on reconcile right? maybe that bucket can just be in "Inf", for this reason I was suggesting 4 min as last bucket. For the rest I am fine with your suggestions, they are more or less the same I did.

Reply:
I think 5 minutes bucket might be interesting ... 5+ minutes would suggests there is a high probability that something is timing out in the reconciliation. I'm fine if we want to add more between 2 and 5 minutes ... but I think the 5 minute is significant as the timeout and should be there.

Reply:
but the bucket should be "less equal" so the 5 min bucket will be filled with values between 2m:01sec to 4m:59sec, everything above 5 min will be in Inf bucket.

Reply:
Yeah, exactly ... everything in the bucket above 5 minutes is suspicious. 2-5 minutes is is long, but not necessarily suspicious since it means that for sure nothing timed out.

Reply:
Ok so just to clarify it will be interesting to check the "Inf" bucket not the "5m" bucket :-)
Anyway, I am fine with yours suggestions.

Reply:
Right ... I think the 5m-Inf bucket is more useful than 4m-Inf bucket.